### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by sauerteig**

### CODE REVIEW

File: B827EBFFFEE2ACB5.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "servers": [
+      {
+        "server_address": "router.eu.thethings.network",
+        "serv_port_up": 1700,
+        "serv_port_down": 1700,
+        "serv_enabled": true
+      }
+    ],
+    "ref_latitude": 48.95463,
+    "ref_longitude": 8.84745
```
Comment (gonzalocasas) [inline B827EBFFFEE2ACB5.json]: you're missing a comma here

**Comment by gonzalocasas** [comment]
@dylancollaud it's cool that you're helping with merging PRs, but please make sure you check them first. ;) This one is actually broken, there's a missing comma in the file, I will fix it on master directly.

**Comment by sauerteig** [comment]
@gonzalocasas  @dylancollaud thanks for your support but there was an other mistake: typo error in the "ref_longitude". Correct value is 8.4745. I corrected the .json at mit github. Does this correction reach you sometime?

**Comment by gonzalocasas** [comment]
@sauerteig I don't see the change anyway. Could you check if you pushed?

**Comment by sauerteig** [comment]
@gonzalocasas Einfach noch leichte Verständnisprobleme mit den Funktionen von Github...

**Pull request merged**